web/src/test/java/org/openmrs/module/web/WebModuleUtilTest.java

```diff
@@ -128,6 +128,24 @@ public void startModule_shouldCreateDwrModulesXmlIfNotExists() throws Exception
 		
 		ModuleFactory.getStartedModulesMap().clear();
 	}
+
+	@Test(expected = ModuleException.class)
+	public void startModule_shouldnotThrowModuleExceptionWhenSpacesInDirectonryName() throws Exception {
+		// create dummy module and start it
+		Module mod = buildModuleForMessageTest();
+		ModuleFactory.getStartedModulesMap().put(mod.getModuleId(), mod);
+		
+		ServletContext servletContext = mock(ServletContext.class);
```

Comment:
Your method name says should NOT throw module exception. But your test expects it. Do you notice this contradiction?